website/content/en/v0.36/upgrading/upgrade-guide.md

```diff
@@ -25,14 +25,6 @@ If you get the error `invalid ownership metadata; label validation error:` while
 
 * As part of the helm chart [karpenter](https://gallery.ecr.aws/karpenter/karpenter) - [source](https://github.com/aws/karpenter/blob/main/charts/karpenter/crds). Helm [does not manage the lifecycle of CRDs using this method](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/), the tool will only install the CRD during the first installation of the Helm chart. Subsequent chart upgrades will not add or remove CRDs, even if the CRDs have changed. When CRDs are changed, we will make a note in the version's upgrade guide.
 
-In general, you can reapply the CRDs in the `crds` directory of the Karpenter Helm chart:
```

Comment:
Why did we drop this bit?

Reply:
It was one AI we had but I think now that we are enabling the webhooks by default it isn't as risky to leave in the guide. I'll put it back.